src/app/timer/timer.component.ts

```diff
@@ -1,8 +1,25 @@
-import { Component } from '@angular/core';
+import { GameManagerService } from './../game-manager/game-manager.service';
+import { TimerService } from './../shared/timer/timer.service';
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
 
 @Component({
   selector: 'app-timer',
   templateUrl: './timer.component.html',
   styleUrls: ['./timer.component.css'],
 })
-export class TimerComponent {}
+export class TimerComponent implements OnInit {
+  timeData?: Observable<{ passed: number; left: number }>;
+  isGameStarted?: Observable<boolean>;
+
+  constructor(
+    private timerService: TimerService,
+    private gameManager: GameManagerService,
+  ) {}
+
+  ngOnInit(): void {
+    this.gameManager.refreshCurrentGameInfo();
```

Comment:
you should not refresh current game info from timer. please do it within Page component

Reply:
done